Notify entity platform (#32170)
home-assistant/home-assistant.github.io: source/_integrations/notify.markdown

One notification integration is automatically included, the Persistent Notifications which creates a notification in the sidebar of the web interface of Home Assistant. This can be chosen with the action named "Notifications: Send a persistent notification" which uses the service `notify.persistent_notification`.

## Notify entity service

> Integrations can also implement the notify entity platform. Entity platform implementations will replace the legacy notify service in time. There is an entity platform service `sens_message` to allow you to send notification messages to multiple notify entities.

Reviewer: ```suggestion
Integrations can also implement the notify entity platform. Entity platform implementations will replace the legacy notify service in time. There is an entity platform service `send_message` which allows you to send notification messages to multiple notify entities.
```

| Service data attribute | Optional | Description |
| ---------------------- | -------- | ----------- |
| `message`              |       no | Body of the notification.

## Companion app notifications

> A common notification integration is via the companion app for Android or iPhone. This can be chosen with the action "Send a notification via mobile_app_your_phone_name" which uses the service `notify.mobile_app_your_phone_name`. See the [companion app documentation](https://companion.home-assistant.io/docs/notifications/notifications-basic) for lots of customization options.

Reviewer: ```suggestion
A common notification integration is via the Home Assistant Companion app for Android or iPhone. This can be chosen with the action "Send a notification via mobile_app_your_phone_name", which uses the service `notify.mobile_app_your_phone_name`. Refer to the [Companion app documentation](https://companion.home-assistant.io/docs/notifications/notifications-basic) for lots of customization options.
```

After you setup a [notifier](/integrations/#notifications), a simple way to test if you have set up your notify platform correctly is to open {% my developer_services title="**Developer tools** > **Services**" %}** tab from the sidebar. Choose your service from the **Service** dropdown menu depending on the integration you want to test, such as **Notifications: Send a persistent notification** or **Notifications: Send a notification via mobile_app_your_phone_name**. Enter your message into the **message** field, and select the **CALL SERVICE** button.

> To test the entity platform service select the `notify.send_message` service, select one or more of `entity`, `device`, `area` or `label` and supply a `message`.

Reviewer: ```suggestion
To test the entity platform service, select the `notify.send_message` service, and select one or more of `entity`, `device`, `area`, or `label`. Then, supply a `message`.
```

### Example with the entity platform notify service

> In the **Developer Tools**, on the **Services** tab, select the **Notifications: Send a notification message** action. Select some target entity's using the entity selectors, enter a message and test sending it.

Reviewer: ```suggestion
In the **Developer Tools**, on the **Services** tab, select the **Notifications: Send a notification message** action. Select some target entities using the entity selectors, enter a message and test sending it.
```

> In the **Developer Tools**, on the **Services** tab, select the **Notifications: Send a notification message** action. Select some target entity's using the entity selectors, enter a message and test sending it.

Reviewer: ```suggestion
Under {% my developer_services title="**Developer Tools** > **Services**" %}, select the **Notifications: Send a notification message** action. Select some target entity's using the entity selectors, enter a message and test sending it.
```

> If you switch to view the YAML data under **Developer Tools**, it will appear as below. The same {% term action %} can be chosen in {% term automation %} actions %, whose YAML will appear the same:

Reviewer: ```suggestion
If you switch to view the YAML data under **Developer Tools**, it will appear as below. The same {% term action %} can be chosen in {% term automation %}. The YAML will appear the same:
```
